dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -54,6 +54,8 @@
 @Path("/v1/workflow")
 public class WorkflowResource {
 
+    public static final String OK = "Ok";
```

Comment:
I would prefer this on the ResponseView so someone else can use it

Reply:
ok. That makes sense